server/controllers/eventController.js

```diff
@@ -80,12 +80,17 @@ exports.createEvent = async (req, res) => {
     description,
     location,
     datetime,
-    category,
     max_participants,
     min_age,
-    price
+    price,
+    category
   } = req.body;
-  const cover = req.file || null;
+  let cover;
+  if (req.file) {
+    cover = req.file.path;
```

Comment:
Look on 154 row, you should do the same or path will be faulty

Reply:
I will fix it, but is it correct when you will write the path into DB with a static URL?

Reply:
Done